### DISCUSSION TITLE
Fork(s) of another's project/repos..

### DISCUSSION THREAD

**Discussion by vandiel01** [community/community#37984] [Other Feature Feedback, Questions, & Ideas]
I do check Forks sometimes of another person, with credit due.. No issue with that.  Only thing I wish to see is an "date" of that forks' last update so we don't have to hop on to that person's fork/repo and check if its updated.  Something like this for example:

![image](https://user-images.githubusercontent.com/52209920/199261158-06c66926-426c-45fc-b58b-8efb35fb99da.png)

Hope it can be done :D

**Comment by ossanna16** [comment]
Hi there @vandiel01 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jfullstackdev** [comment]
why would you be needing that ? forks are not the single source of truth in development, it is the `main` that every developer should focus

anyways, that can still be a valid request, but you know, on top of all those that needs to be implemented first, this might not be truly a priority

**Reply by vandiel01** [reply]
some author may have quit and another takes place but with respect of original author (with proper Licence type, etc), yet fork would have an new updated code.  Hence is why those "last update" would be a good idea to see if the original had made any change or one of the forks had...

**Comment by Abdalhalanz45** [comment]
> I do check Forks sometimes of another person, with credit due.. No issue with that.  Only thing I wish to see is an "date" of that forks' last update so we don't have to hop on to that person's fork/repo and check if its updated.  Something like this for example:
> 
> ![image](https://user-images.githubusercontent.com/52209920/199261158-06c66926-426c-45fc-b58b-8efb35fb99da.png)
> 
> Hope it can be done :D
